diff --git a/llama-index-integrations/vector_stores/llama-index-vector-stores-mongodb/pyproject.toml b/llama-index-integrations/vector_stores/llama-index-vector-stores-mongodb/pyproject.toml
index ba22a585c..f7b6c5647 100644
--- a/llama-index-integrations/vector_stores/llama-index-vector-stores-mongodb/pyproject.toml
+++ b/llama-index-integrations/vector_stores/llama-index-vector-stores-mongodb/pyproject.toml
@@ -41,6 +41,7 @@ jupyter = "^1.0.0"
 llama-index-embeddings-openai = "^0.3.0"
 llama-index-llms-openai = "^0.3.0"
 llama-index-readers-file = "^0.4.0"
+llama-index-core = "^0.12.1"
 mypy = "0.991"
 pre-commit = "3.2.0"
 pylint = "2.15.10"
